src/coordinator/index.spec.js

```diff
@@ -391,23 +391,34 @@ describe('Coordinator', function() {
   describe('getUsersByOrganization', function() {
     context('the organization ID is provided', function() {
       let expectedOrganizationId;
-      let organizationUsers;
+      let expectedOrganizationUsers;
+      let organizationUsersFromServer;
       let promise;
       let request;
+      let toCamelCase;
 
       beforeEach(function() {
         expectedOrganizationId = faker.random.uuid();
 
-        organizationUsers = fixture.buildList(
+        expectedOrganizationUsers = fixture.buildList(
           'contxtUser',
-          faker.random.number({ min: 1, max: 10 })
+          faker.random.number({ min: 1, max: 10 }),
+          { id: expectedOrganizationId }
```

Comment:
This should just be `expectedOrganizationUsers = fixture.buildList('contxtUser', faker.random.number(min: 1, max: 10 }));` as the user IDs shouldn't match the organization ID